packages/gatsby-cli/src/util/__mocks__/gatsby-core-utils.js

```diff
@@ -0,0 +1 @@
+module.exports = {}
```

Comment:
I think this might be causing lots of failing tests right now (didn't have time to investigate - just looking at CI checks on the PR)

Reply:
I just deleted these mocks, I thought they were necessary but found out that running `yarn lerna-prepare` helped: the modules were missing because they were not built! :smile: